### PR TITLE
[APT-1931] Added new error pages

### PR DESCRIPTION
**Pull request by eak12913**
These pages will be used for the checkout process. In case of customer errors - we will redirect to one of these two pages.

### CODE REVIEW

File: pages/payment-incomplete/index.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        <div class="col-xs-12 text-center">
+          <h1>{{ page.title }}</h1>
+          <p>{{ page.excerpt }}</p>
+          <p>
+            <a
+              href="#"
+              onclick="window.history.back()"
+              class="btn btn-info"
+              role="button"
+              >Go Back</a
```
Comment (ebeneliason) [inline pages/payment-incomplete/index.html:21]:
```suggestion
              >Back to Checkout</a
```

File: pages/payment-incomplete/index.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+---
+
+<div class="main">
+  <div class="section section-hero">
+    <div class="container">
+      <div class="row">
+        <div class="col-xs-12 text-center">
+          <h1>{{ page.title }}</h1>
+          <p>{{ page.excerpt }}</p>
+          <p>
```
Comment (ebeneliason) [inline pages/payment-incomplete/index.html:15]: I think this extra `<p>` wrapper is unnecessary.

File: pages/charge-failed/index.html [hunk 3]
```diff
@@ -0,0 +1,25 @@
+---
+layout: default
+title: Payment Incomplete
+excerpt: We were unable to process your payment. A member of the Gruntwork sales team will reach out soon to coordinate payment and activate your subscription.
+permalink: /charge-failed/
+slug: charge-failed
```
Comment (ebeneliason) [inline pages/charge-failed/index.html]:
```suggestion
```

File: pages/payment-incomplete/index.html [hunk 4]
```diff
@@ -0,0 +1,29 @@
+---
+layout: default
+title: Payment Incomplete
+excerpt: We were unable to process your payment. Please ensure you've entered your payment information correctly, or reach out to <a href="mailto:sales@gruntwork.io">sales@gruntwork.io</a> if you need assistance.
+permalink: /payment-incomplete/
+slug: payment-incomplete
```
Comment (ebeneliason) [inline pages/payment-incomplete/index.html]:
```suggestion
```

File: pages/payment-incomplete/index.html (hunk 1 above)
Comment (eak12913) [reply pages/payment-incomplete/index.html:21]: ✅

File: pages/payment-incomplete/index.html (hunk 2 above)
Comment (eak12913) [reply pages/payment-incomplete/index.html:15]: ✅

**Review by ebeneliason — APPROVED** [review]
Changes look good!

**Pull request merged**